.gitignore

```diff
@@ -1,2 +1,6 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
+
+news-controller_copy.js
```

Comment:
what are these copy js files?

Reply:
for me to work on the files whilst waiting for review so then i know which code line my test file was on...

Reply:
I made a duplicate of my current files so that i may work on them